### PR TITLE
feat: select character in CharacterSelectorScreen with hotkey

### PR DESCRIPTION
**Pull request by Mcrtin**

### CODE REVIEW

**Comment by magicus** [comment]
Hi, you seem to be running some kind of IDE plugin that keeps spamming me about code reviews. Can you please turn that off, it is very annoying.

**Comment by Mcrtin** [comment]
Of course, I'm so sorry.

File: common/src/main/java/com/wynntils/screens/characterselector/CharacterSelectorScreen.java [hunk 1]
```diff
@@ -216,6 +218,18 @@ public boolean mouseReleased(double mouseX, double mouseY, int button) {
...
 
+    @Override
+    public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
+        KeyMapping[] keyHotbarSlots = McUtils.options().keyHotbarSlots;
+        for (int i = 0; i < keyHotbarSlots.length && i < classInfoList.size(); i++) {
+            if (!keyHotbarSlots[i].matches(keyCode, scanCode)) continue;
+            int slot = classInfoList.get(i).slot();
+            Models.CharacterSelection.playWithCharacter(slot);
+            return true;
+        }
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/screens/characterselector/CharacterSelectorScreen.java]:
```suggestion

        for (int i = 0; i < Math.Min(keyHotbarSlots.length, classInfoList.size()); i++) {
            if (!keyHotbarSlots[i].matches(keyCode, scanCode)) continue;
            int slot = classInfoList.get(i).slot();
            Models.CharacterSelection.playWithCharacter(slot);
            return true;
        }
        
```

File: common/src/main/java/com/wynntils/screens/characterselector/CharacterSelectorScreen.java [hunk 2]
```diff
@@ -216,6 +218,20 @@ public boolean mouseReleased(double mouseX, double mouseY, int button) {
         return true;
     }
 
+    @Override
+    public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
+        KeyMapping[] keyHotbarSlots = McUtils.options().keyHotbarSlots;
+
+        for (int i = 0; i < Math.Min(keyHotbarSlots.length, classInfoList.size()); i++) {
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/screens/characterselector/CharacterSelectorScreen.java]:
```suggestion
        for (int i = 0; i < Math.min(keyHotbarSlots.length, classInfoList.size()); i++) {
```

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**